neo/IO/Data/LevelDB/WriteOptions.cs

```diff
@@ -5,6 +5,8 @@ namespace Neo.IO.Data.LevelDB
     public class WriteOptions
     {
         public static readonly WriteOptions Default = new WriteOptions();
+        public static readonly WriteOptions DefaultSync = new WriteOptions() { Sync = true };
```

Comment:
```suggestion
        public static readonly WriteOptions Sync = new WriteOptions() { Sync = true };
```

Reply:
Conflicts with the property of this object

Reply:
```suggestion
        public static readonly WriteOptions SyncWrite = new WriteOptions() { Sync = true };
```